content/blog/2020/2020-01-29-machine-anthropology.md

```diff
@@ -0,0 +1,44 @@
++++
+author = "Ajda Pretnar"
+date = "2020-01-29"
+draft = false
+title = "What is Machine Anthropology?"
+type = "blog"
+thumbImage = "/blog_img/2020/2020-01-29-MA-small.png"
+frontPageImage = "/blog_img/2020/2020-01-29-MA-small.png"
+categories = ["machine", "anthropology", "big data", "pivot table"]
+shortExcerpt = "At the recent Machine Anthropology workshop we used Orange to explore anthropological data."
+longExcerpt = "At the recent Machine Anthropology workshop we used Orange to explore anthropological data."
++++
+
+For those unfamiliar with the field, cultural anthropology is the study of human cultures, practices and habits in a holistic and comparative manner. Its core method is ethnographic fieldwork, which means researchers spend a long time in the field with their subjects, live with them, talk with them, socialize with them, and observe relationships and behaviours. But recently, anthropology has begun to use also machine learning and data mining as a part of its method. The subdiscipline is called computational anthropology (combining ethnographic fieldwork *with* big data) or machine anthropology (ethnography *as* big data).
+
+Related: {{< link_new url="hblog/2018/11/06/data-mining-for-anthropologists/" name="Data Mining for Anthropologists">}}
+
+{{< figure src="/blog_img/2020/2020-01-29-MA-big.png" >}}
+
+Copenhagen Center for Social Data Science ([SODAS](https://sodas.ku.dk/)) held a conference on machine anthropology just a few days ago and it was inspiring to see how the humanities benefit from computer sciences and vice versa! In our talk I presented my PhD research of sensor data from a smart building and used Orange to show how to detect patterns of behaviour in such data.
```

Comment:
You can skip "In our talk" to avoid the mid-sentence jump from plural to singular.

"ago and it" -> Consider splitting into two sentences --- "ago. It"